server/migrations/20210302210535_create_users_table.js

```diff
@@ -0,0 +1,11 @@
+exports.up = function(knex) {
+  return knex.schema.createTable('users', t => {
+    t.uuid('id')
+    t.string('email').notNullable().unique()
+    t.string('password_hash').notNullable()
+  })
+}
+
+exports.down = function(knex) {
+  return knex.schema.dropTable('users')
+}
```

Comment:
Migrations will be explained in the session.
However, Try to guess what this does :D